.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -917,14 +917,13 @@ stages:
         dependsOn: [ ]
         timeoutInMinutes: 180
         steps:
-          # TODO(artunduman): Remove 'daily' when available
           - bash: |
               echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
               echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
               echo '##vso[task.setvariable variable=OS_VERSION]24.04'
               echo '##vso[task.setvariable variable=IMG_PUBLISHER]Canonical'
-              echo '##vso[task.setvariable variable=IMG_OFFER]0001-com-ubuntu-server-noble-daily'
-              echo '##vso[task.setvariable variable=IMG_SKU]24_04-daily-lts-arm64'
+              echo '##vso[task.setvariable variable=IMG_OFFER]ubuntu-24_04-lts'
```

Comment:
make sure the ARM64 24.04 build pipeline still works.

Reply:
it does, I attached the build link to description

Reply:
also thanks to Alison resolving the previous scanning issue in her PR
https://github.com/Azure/AgentBaker/pull/4803